package.json

```
@@ -19,9 +20,8 @@
     "typescript": "^4.2.2"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.20",
-    "jest": "^26.6.3",
+    "@vitest/ui": "^0.18.1",
     "prettier": "^2.2.1",
-    "ts-jest": "^26.5.2"
+    "vitest": "^0.18.1"
```

I confirmed the latest `vitest` works successfully.

```suggestion
    "vitest": "^0.25.1"
```